rusoto/services/autoscaling/README.md

```diff
@@ -12,7 +12,7 @@ You may be looking for:
 ## Requirements
 
 Rust stable or beta are required to use Rusoto. Nightly is tested, but not guaranteed to be supported. Older
-versions _may_ be supported. The currently supported Rust versions can be found in the Rusoto project 
```

Comment:
When I check out this branch and run crate generation, it says the whitespace here is being *added* from this PR, which would mean no change when merged back into master.

Question: are stable `rustfmt` and stable `rustc` being used? That's what we're using now, which I think changed since the last PR you made. I had to switch to it for the 0.36.0 release the other day.